BusinessLogicLayer/Initializer/Implementation/Initialize.cs

```diff
@@ -0,0 +1,123 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using BusinessLogicLayer.Initializer.Interfaces;
+using BusinessLogicLayer.ViewModels;
+using DataAccessLayer.DataBaseModels;
+using Microsoft.EntityFrameworkCore.Metadata;
+using OfficeOpenXml;
+
+namespace BusinessLogicLayer.Initializer.Implementation
+{
+    public class Initialize : IIInitialize
+    {
+        private UnitOfWork _unitOfWork = new UnitOfWork();
+
+        public string CreatePath(string directoryName = "", string fileName = "Languages")
+        {
+            string rootPath = Directory.GetCurrentDirectory() + @"\wwwroot\Dictionary\" + directoryName + fileName + ".xlsx";
+            Console.WriteLine(rootPath);
+            return rootPath;
+        }
+
+        public ExcelWorksheet GetDataFromFile(string path)
+        {
+            FileInfo fileInfo = new FileInfo(path);
```

Comment:
Please add validation (if file exists) and try-catch (because file may be corrupted or protected).